contract_digitized_signature/i18n/es.po

```diff
@@ -0,0 +1,36 @@
+# Translation of Odoo Server.
+# This file contains the translation of the following modules:
+#	* contract_digitized_signature
+#
+msgid ""
+msgstr ""
+"Project-Id-Version: Odoo Server 9.0c\n"
```

Comment:
Should this file be removed?

Reply:
No, translations format hasn't changed on v10